lib/components/molecules/ArticleThumbnail/types/index.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
```

Comment:
Please define all prop data types and mark them required/optional

Reply:
Done